package.json

```diff
@@ -58,7 +58,7 @@
     "@types/lodash": "^4.14.191",
     "@types/mocha": "^5.2.7",
     "@types/node": "^14.14.31",
-    "@types/node-fetch": "^2.6.4",
```

Comment:
you changed `@types/node-fetch` instead of `node-fetch`

Reply:
I've fixed it. Sorry its Friday)